File: java/level-0/VariableExchange.java
import java.util.Scanner;

public class VariableExchange {
    public static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) {
        variableExchange1();
        variableExchange2();
    }

    public static void variableExchange1() {
        String favouriteDrink1 = "Milk";
        String favouriteDrink2 = "Juice";
        String favouriteDrinkTemp = "";

        System.out.println("Drink Favourite 1 Before : " + favouriteDrink1);
        System.out.println("Drink Favourite 2 Before : " + favouriteDrink2);

        favouriteDrinkTemp = favouriteDrink1;
        favouriteDrink1 = favouriteDrink2;
        favouriteDrink2 = favouriteDrinkTemp;

        System.out.println("Drink Favourite 1 After : " + favouriteDrink1);
        System.out.println("Drink Favourite 2 After : " + favouriteDrink2);
    }

    public static void variableExchange2() {
        // Just For Numeric
        int a = 25;
        int b = 39;

        System.out.println("Value of a before : " + a);
        System.out.println("Value of b before : " + b);

        a = a + b;
        b = a - b;
        a = a - b;

        System.out.println("Value of a after : " + a);
        System.out.println("Value of b after : " + b);
    }
}

File: javaScript/level-0/variable-exchange.js
let favouriteProgrammingLanguage1 = "Java";
let favouriteProgrammingLanguage2 = "JavaScript";
let favouriteProgrammingLanguageTemp = "";

let a = 20;
let b = 57;

function variableExchange1() {
  console.log(
    "Favourite Programming Language 1 Before : " + favouriteProgrammingLanguage1
  );
  console.log(
    "Favourite Programming Language 2 Before : " + favouriteProgrammingLanguage2
  );

  favouriteProgrammingLanguageTemp = favouriteProgrammingLanguage1;
  favouriteProgrammingLanguage1 = favouriteProgrammingLanguage2;
  favouriteProgrammingLanguage2 = favouriteProgrammingLanguageTemp;

  console.log(
    "Favourite Programming Language 1 After : " + favouriteProgrammingLanguage1
  );
  console.log(
    "Favourite Programming Language 2 After : " + favouriteProgrammingLanguage2
  );
}

function variableExchange2() {
  console.log("Value of a before : " + a);
  console.log("Value of b before : " + b);

  a = a + b;
  b = a - b;
  a = a - b;

  console.log("Value of a after : " + a);
  console.log("Value of b after  : " + b);
}

variableExchange1();
variableExchange2();
